modules/manage/pages/cluster-maintenance/disk-utilization.adoc

```diff
@@ -237,7 +237,7 @@ Space management divides the disk space into different areas that can be managed
 * Reserved disk space (`disk_reservation_percent`) is the area of disk space that Redpanda does not use.
 ** As the disk space used by the other areas grows to their target sizes, the reserved space provides buffer space to avoid free disk space alerts.
 ** SSDs that run near capacity can experience performance degradation, so this buffer space prevents the disks from running at capacity.
-* Cache storage (the minimum of `cloud_storage_cache_size_percent` or `cloud_storage_cache_size`) is the maximum size of the xref:../tiered-storage.adoc#caching[disk cache] used by Tiered Storage. As the cache reaches its limit, new data added to the cache removes old data from the cache.
+* Cache storage (`cloud_storage_cache_size_percent`, or `cloud_storage_cache_size` which overrides the former) is the maximum size of the xref:../tiered-storage.adoc#caching[disk cache] used by Tiered Storage. As the cache reaches its limit, new data added to the cache removes old data from the cache.
```

Comment:
This isn't quite accurate: if both are set, Redpanda will honor the minimum of the two